src/main.rs

```diff
@@ -80,7 +90,53 @@ fn main() {
         Rank::InfoPlusProbability => roget::Rank::InfoPlusProbability,
         Rank::ExpectedInformation => roget::Rank::ExpectedInformation,
     };
-    play(move || solver.build(), args.games);
+    if args.interactive {
+        play_interactive(move || solver.build());
+    } else {
+        play(move || solver.build(), args.games);
+    }
+}
+
+fn play_interactive<G>(mk: impl FnOnce() -> G)
+where
+    G: Guesser,
+{
+    let mut guesser = mk();
+    let mut history = Vec::new();
+    // Wordle only allows six guesses.
+    for _ in 1..=6 {
+        let guess = guesser.guess(&history);
+        let correctness = {
+            println!("Guess:  {}", guess.to_uppercase());
+            print!("Colors: ");
```

Comment:
We should be a bit more helpful here and explain that the user should use C/M/W.

Reply:
Is it enough ?